examples/tree/plot_unveil_tree_structure.py

```diff
@@ -63,6 +65,20 @@
 #   - ``n_node_samples[i]``: the number of training samples reaching node
 #     ``i``
 #   - ``impurity[i]``: the impurity at node ``i``
+#   - ``weighted_n_node_samples[i]``: the weighted number of training samples
+#     reaching node ``i``
+#
```

Comment:
I'll add a short description of `value[i]` here and explain on it after the Tree diagram.

I think it's better to explain what values are at the end of the [Tree structure section](https://scikit-learn.org/stable/auto_examples/tree/plot_unveil_tree_structure.html#tree-structure), after the tree diagram. The tree diagram contains the values, so it is a good place to explain them. I know it is a special case of `n_outputs=1`, but I think it flows better.

Reply:
I've updated the description. Lmk wdyt?